src/amber/cli/templates/controller.cr

```diff
@@ -7,10 +7,9 @@ module Amber::CLI
 
     @name : String
     @actions = Hash(String, String).new
-    @language : String
+    getter language : String = Amber::CLI::Config.language
```

Comment:
Similar comment to above. This is already in the `Amber::CLI` namespace, so it should work with just `Config.language`